css/selectors/cue.json

```
@@ -53,6 +49,54 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "with_parameter": {
```

The spec calls this "the ::cue(selector) pseudo-element with an argument" so I figured we should try to match that terminology:

```suggestion
        "selector_argument": {
```